_data/api-swagger.yaml

```diff
@@ -649,6 +650,51 @@ definitions:
         }
       }
       ```
+
+  Itsme Identification:
+    allOf:
+    - $ref: '#/definitions/Verification'
+    - type: object
+      properties:
+        PhoneNumber:
+          type: string
+          readOnly: true
+          description:             
+            "The mobile phone number of the signer.
+            Must conform to E.164,
+            [the international public telecommunication numbering plan](https://en.wikipedia.org/wiki/E.164),
+            which requires the country calling code (Only the Belgian country calling code is supported: +32)."
```

Comment:
I don't think the openening and closing quote `"` are necessary.

Reply:
The quotes are necessary because of the comma that is used in the text.